CMakePresets.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 7,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 28,
+        "patch": 0
+    },
+    "include": ["./support/Environments/Mbot/MbotPresets.json"]
```

Comment:
Ah this is a nice feature! I was wondering how we might make available the different presets without placing them in the root of the repo.